blockchain/types/chain_hashes.go

```diff
@@ -18,20 +18,32 @@ package types
 
 import (
 	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/ser/rlp"
 	"math/big"
 )
 
 type ChainHashes struct {
+	Type uint8
+	Data []byte
+}
+
+type ChainHashesInternalType0 struct {
```

Comment:
Is there a specific reason for the number '0' after it?

Reply:
It will be expanded type 1, 2 later if needed.